notebooks/config-exp-cookbook.ipynb

```diff
@@ -0,0 +1,276 @@
+   "id": "bc6e47f6-ed03-41ef-bacf-b1790b5bf56f",
+   "metadata": {},
+   "source": [
+    "Here the list of config file paths is created and passed to `configure_experiment()`. Note that if the configs share any keys, the values from files later in the list will override the earlier ones. For example, `convert_nst:` had a `true` value in the base config file, but it was changed to `false` by the HRRR ICs config. While some Jinja 2 expressions were not able to be rendered and still exist here, the keys `data_dir_input_grid:` and `rundir:` have had their Jinja 2 expressions rendered. \n",
+    "<!--cell 10-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e863d5ac-f727-4d91-a4bd-9bf813d35e6c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "user:\n",
+       "  ACCOUNT: zrtrr\n",
+       "  MACHINE: hera\n",
+       "workflow:\n",
+       "  EXPTDIR: /path/to/my/output\n",
+       "task_make_ics:\n",
+       "  chgres_cube:\n",
+       "    namelist:\n",
+       "      update_values:\n",
+       "        config:\n",
+       "          cycle_day: !int '{{ cycle.strftime(''%d'') }}'\n",
+       "          cycle_hour: !int '{{ cycle.strftime(''%H'') }}'\n",
+       "          cycle_mon: !int '{{ cycle.strftime(''%m'') }}'\n",
+       "          convert_atm: true\n",
+       "          convert_nst: false\n",
+       "          convert_sfc: true\n",
+       "          data_dir_input_grid: /path/to/my/output/make_ics\n",
+       "          external_model: HRRR\n",
+       "          input_type: grib2\n",
+       "          tg3_from_soil: true\n",
+       "          tracers: ''\n",
+       "          tracers_input: ''\n",
+       "          varmap_file: '{{ user.PARMdir }}/ufs_utils/varmap_tables/GSDphys_var_map.txt'\n",
+       "          geofrid_file_input_grid: '{{ workflow.FIXgsm }}/geo_em.d01.nc_HRRRX'\n",
+       "          nsoil_out: 9\n",
+       "    rundir: /path/to/my/output/make_ics"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "user_config_files = ['fixtures/config-exp/base-file.yaml',\n",
+    "                     'fixtures/config-exp/hrrr-ics.yaml',\n",
+    "                     'fixtures/config-exp/fv3-rap-physics.yaml',\n",
+    "                     'fixtures/config-exp/user.yaml'\n",
+    "                    ]\n",
+    "configure_experiment(user_config_files)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python [conda env:DEV-uwtools] *",
+   "language": "python",
+   "name": "conda-env-DEV-uwtools-py"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.12.3"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
I had a thought about showing how this experiment config could be used with a driver as a next step and then dismissed it as out-of-scope....and then, I caught a typo in a key.

How would you feel about adding one more example to the end of the notebook that shows how a user might validate the chgres_cube block with the driver validate method?

This is an action that I'd like to add to the SRW configuration layer and is (very lightly) documented in Issue #647, so it might be nice to show a brief example here.

Reply:
I think this is a good example to add, but ultimately validating the config didn't find any typos in any keys. I found one that I fixed (geofrid_file_input_grid -> geogrid_file_input_grid) but let me know if there were any others.

Reply:
That's the only one I saw, too.